dts/arm/st/h7/stm32h7.dtsi

```diff
@@ -861,7 +861,7 @@
 			compatible = "st,stm32-sdmmc";
 			reg = <0x52007000 0x400>;
 			clocks = <&rcc STM32_CLOCK_BUS_AHB3 0x00010000>,
-				 <&rcc STM32_SRC_HSI48 SDMMC_SEL(0)>;
+				 <&rcc STM32_SRC_PLL1_Q SDMMC_SEL(0)>;
```

Comment:
Then, we need to verify that boards enabling sdmmc have a PLLQ producing 48MHz output.

Reply:
for stm32H7x, the sdmmc source clock is not necessarily 48MHz

Reply:
But then, I guess we need to check its not too far neither.

Reply:
there is no constraint on the sdmmc clock, for example 137.5MHz from pll1_q (when sysclock = 550MHz)